src/api/.rubocop_todo.yml

```diff
@@ -27,6 +27,12 @@ Lint/NonLocalExitFromIterator:
     - 'app/controllers/configurations_controller.rb'
     - 'app/models/patchinfo.rb'
 
+# Offense count: 1
+# Cop supports --auto-correct.
```

Comment:
You put a cop with one offense that supports auto correct into the TODO? :wink:

Reply:
okay, yes that does not make too much sense :+1:

Reply:
@hennevogel please have a look again